_posts/2020-01-23-operating-system-and-browser-accessibility-display-modes.md

```diff
@@ -0,0 +1,373 @@
+
+<figure role="figure" aria-label="daverupert.com has support for Dark Mode.">
+  <img alt="A side-by-side comparison of light and dark themes for the home page daverupert.com. The navigation, text, background, and link colors update, but the logo and illustration of Dave as a fantasy warrior remain the same." src="/img/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/dark-mode.png">
+  <figcaption><a href="https://daverupert.com/">daverupert.com</a> has support for Dark Mode.</figcaption>
+</figure>
+
+### How to enable it
+
+Assumes the latest version of each Operating System or browser.
+
+#### Android
+
+1. Go to **Settings**.
+1. Go to **Display**.
+1. Set the **Dark Theme** toggle to the **on** position.
+
+#### iOS
+
+1. Go to **Settings**.
+1. Go to **Display and Brightness**.
+1. Select the **Dark** option for **Appearance**.
+
+#### macOS
+
+1. Go to **System Preferences**.
+1. Go to **General**.
+1. Select the **Dark** option for **Appearance**.
+
+#### Windows
+
+1. Go to **Settings**.
+1. Go to **Personalization**.
+1. Go to **Colors**.
+1. Select the **Dark** option for **Choose your default app mode**.
+
+### How to target it in code
+
+You can use the [`prefers-color-scheme` media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) to to target Dark Mode in CSS. In the following example, we set our web page to default to black text on a white background, but update to be white text on a black background when Dark Mode is enabled:
+
+``` css
+body {
+  background-color: #ffffff;
+  color: #000000;
+}
+
+@media (prefers-color-scheme: dark) {
+  body {
+    background-color: #000000;
+    color: #ffffff;
+  }
+}
+```
+
+## Increased Contrast Mode
+
+This mode is only available on macOS and iOS. It will increase the difference in color values between the gray colors used for the OS visuals, as well as drawing more obvious borders between different UI elements.
+
+This is helpful for:
+
+- People experiencing low vision conditions,
+- and/or who have lower quality displays,
+- and/or less technologically literate people, who may need the difference between different parts of the OS visuals to be more obvious.
+
+An example of one of these conditions could be vision-related issues caused by Glaucoma. Here, the increased contrast and borders may help a person who has blurry central vision and a loss of peripheral vision understand the boundaries of different parts of the interface.
+
+### What it affects
+
+Increased Contrast Mode will update **all** User Interface (<abbr>UI</abbr>) elements of macOS and iOS. Most apps developed by Apple will also support this mode. Third party apps that use Apple’s interface library will also support this mode, but custom, nonstandard UI elements will **not** be supported. Websites and web apps do **not** support this mode.
```

Comment:
you already abbreviated UI.  

no problem if you want to keep it in under the assumption people may skip to particular headings/sections of content.  but just figured i should mention that.